Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -76,8 +77,20 @@ extension Snapshotting where Value == URLRequest, Format == String {
     }
 
     // URL
-    components.append("\"\(request.url!.absoluteString)\"")
+    if let url = request.url?.sortingQueryItems() {
+      components.append("\"\(url.absoluteString)\"")
+    }
```

Comment:
Force unwrapping here is fine:

```suggestion
    components.append("\"\(request.url!.absoluteString)\"")
```